src/Sylius/Behat/Context/Setup/OrderContext.php

```diff
@@ -349,6 +360,42 @@ public function customersHavePlacedOrdersForTotalOf($numberOfCustomers, $numberO
     }
 
     /**
+     * @Given the customer canceled this order
+     */
+    public function theCustomerCanceledThisOrder()
+    {
+        $this->applyTransition('cancel');
+    }
+
+    /**
+     * @Given the customer confirmed this order
+     */
+    public function theCustomerConfirmedThisOrder()
+    {
+        $this->applyTransition('confirm');
+    }
+
+    /**
+     * @param string $transition
+     *
+     * @throws InvalidTransitionException
+     */
+    private function applyTransition($transition)
+    {
+        /** @var OrderInterface $order */
+        $order = $this->sharedStorage->get('order');
```

Comment:
IMO we should pass this object in method argument or change method name to ``applyOrderTransition``.